### PR TITLE
Added Cloud Native Leaders Section

### LINKED ISSUE #1422: Implementation of the Cloud Native Leaders section on homepage
**Current Behavior**
We currently have no implementation of the Cloud Native Leaders section on homepage 

**Desired Situation**
The design of the home page is currently approved on figma, and we will appreciate someone jumping on the task of implementing it 

https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=1869%3A2967

Checkout the design below for easy navigation and identification
cloud

[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by chinmaym07**
Signed-off-by: chinmaym07 <b418020@iiit-bh.ac.in>

**Description**

This PR fixes #1422 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Cloud-Native/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+              From the beginning Layer5 actively engaged in leadership roles in CNCF initiatives and projects.Commissioned by the CNCF for research and analysis.
+            <br />
+              Published the Serverless Whitepaper Chatir of 
+          </p>
+          <img src={cncf} alt="Cloud Native-logo" />
+        </div>
+        
+      </div>
+      <div className="btn-cont">
+        <Button primary title="About Us" url="" />
```
Comment (Tanuj22) [inline src/sections/Cloud-Native/index.js:26]: Link - `/company/about/`

**Comment by Tanuj22** [comment]
Looking great. Adding the link we are good to go!

File: src/sections/Cloud-Native/index.js (hunk 1 above)
Comment (chinmaym07) [reply src/sections/Cloud-Native/index.js:26]: Thanks for the review ...Sorry I forgot to add that .

Comment (chinmaym07) [reply src/sections/Cloud-Native/index.js]: Fixed.!

**Review by Jashpatel1 — APPROVED** [review]
Looks good. Thanks @chinmaym07

**Pull request merged**